types/ldap-escape/ldap-escape-tests.ts

```diff
@@ -0,0 +1,16 @@
+/* tslint:disable:no-namespace */
+'use strict';
+
+import { dn, filter } from 'ldap-escape';
+
+namespace Module {
```

Comment:
I also don't think you need this namespace here, or the `/*tslint:disable:no-namespace */` line if you get rid of the namespace.